wp-admin/js/widgets/media-audio-widget.js

```diff
@@ -82,11 +113,12 @@
 			metadata = control.mapModelToMediaFrameProps( control.model.toJSON() );
 
 			// Set up the media frame.
-			mediaFrame = wp.media({
+			mediaFrame = new AudioDetailsMediaFrame({
```

Comment:
Ugh that is too bad the other approaches don't work.  It would be nice if we could also pass an `option` for `states` to the `wp.media` constructor too.